flexmeasures/ui/tests/test_color_defaults.py

```diff
@@ -0,0 +1,12 @@
+from flexmeasures.ui.utils.color_defaults import darken_color, lighten_color, rgba_color
+
+
+def test_color_utils() -> None:
+    primary_color: str = "#1a3443"
+    secondary_color: str = "f1a122"
+
+    assert darken_color(primary_color, 7) == "#18303e"
+    assert darken_color(primary_color, 4) == "#183140"
+    assert rgba_color(primary_color, 0.2) == "rgba(26, 52, 67, 0.2)"
+    assert lighten_color(secondary_color, 4) == "f1a42a"
```

Comment:
So our actual secondary hover color is "ff9d00", and in `get_color_settings()` you also get it like this (lighten by 4).

Did you not aim to re-create the colors we were using before? Not to say that I would have noticed, but I am also not a designer :)

Reply:
Yes that was the aim but i could not get it perfectly but it was just really close.